### PR TITLE
Introduce global settings

### PR DESCRIPTION
**Pull request by maciaszczykm**
- [x] add settings manager
- [x] add global settings endpoint
- [x] add settings view
- [x] add settings menu entry
- [x] handle concurrent settings change error (dialog with save anyways and refresh options)
- [x] merge settings during save to avoid any data loss
- [x] fix tests
- [x] add tests
- [x] create service to retrieve specific settings values and use it across the app
- [x] generate translations
- [x] service refresh after settings save
- [x] handle cluster name setting
- [x] handle items per page setting
- [x] remove bower leftovers
- [x] fix caching problem

### CODE REVIEW

**Comment by maciaszczykm** [comment]
Current progress:

image

**Comment by maciaszczykm** [comment]
Current progress:

image

**Comment by maciaszczykm** [comment]
It is ready for an initial review. I have to fix few things in tests/annotations etc.

File: src/app/backend/settings/manager.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	}
+
+	return s
+}
+
+// GetGlobalSettings implements SettingsManager interface. Check it for more information.
+func (sm *SettingsManager) SaveGlobalSettings(client kubernetes.Interface, s *api.Settings) error {
+	cm, isDiff := sm.load(client)
+	if isDiff {
+		return errors.New("settings changed since last reload")
```
Comment (floreks) [inline src/app/backend/settings/manager.go]: Use `LocalizerError` method here so we can localize it in frontend.

Comment (maciaszczykm) [reply src/app/backend/settings/manager.go]: It is not displayed to the user but used to display "save anyways" dialog.

Comment (maciaszczykm) [reply src/app/backend/settings/manager.go]: I will comment it in the code.

**Comment by floreks** [comment]
Overall LGTM.

File: src/app/backend/settings/manager.go (hunk 1 above)
Comment (maciaszczykm) [reply src/app/backend/settings/manager.go]: Done.

**Comment by maciaszczykm** [comment]
PTAL

**Pull request merged**